Packages/com.unity.inputsystem/InputSystem/Plugins/iOS/iOSSupport.cs

```diff
@@ -41,6 +43,16 @@ public static void Initialize()
                 new InputDeviceMatcher()
                     .WithInterface("iOS")
                     .WithDeviceClass("LinearAcceleration"));
+
+            InputSystem.RegisterLayout<iOSStepCounter>();
+
+            // Don't add devices for InputTestRuntime
+            // TODO: Maybe there should be a better place for adding device from C#
+            if (InputSystem.s_Manager.m_Runtime is NativeInputRuntime)
```

Comment:
Yea it seems suboptimal to do jumpyloops with checking which runtime we're using ... not quite sure what would be a better way to do it tbh.